packages/ember/tailwind-vite/index.test.ts

```diff
@@ -0,0 +1,23 @@
+import { apply, diff, diffSummary, newEmberApp } from "ember-apply/test-utils";
+import { describe, expect, it } from "vitest";
+
+import { default as tailwindVite } from "./index.js";
+
+describe("tailwind-vite", () => {
+  it("default export exists", () => {
+    expect(typeof tailwindVite).toEqual("function");
+  });
+
+  describe("applying to an ember app", () => {
+    it("works via CLI", async () => {
+      let appLocation = await newEmberApp();
```

Comment:
For testing against the new v2 app, i think  we may need a new fenction to call that sets the blueprint flag?

Reply:
I think this is a mistake? If you're referring to the snapshot moving towards the new `@embroider/app-blueprint` that was not intentional as per [this conversation](https://github.com/NullVoxPopuli/ember-apply/issues/587#issuecomment-2610919725).